src/GusApi/Environment/ProdEnvironment.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace GusApi\Environment;
+
+class ProdEnvironment implements EnvironmentInterface
+{
+    /**
+     * @return string
```

Comment:
Nie ma potrzeby tworzenia tego PHPDoca - dla PHP 7.1 nie wnosi on żadnej informacji. Jeżeli jednak wolisz pozostawić je w takich przypadkach, to w tym miejscu powinien znajdować się `@inheritDoc`.